source/add-existing.txt

```diff
@@ -0,0 +1,232 @@
+application that uses {+odm+}, see one of the following guides:
+
+- :ref:`mongoid-quick-start-rails`
+- :ref:`mongoid-quick-start-sinatra`
+
+Sinatra Application
+-------------------
+
+To start using {+odm+} in an existing Sinatra application, you can follow
+the same steps described in the
+:ref:`Sinatra Quick Start <mongoid-quick-start-sinatra>` guide.
+
+The following steps describe how to add {+odm+} to a Sinatra application:
+
+1. Add the ``mongoid`` dependency to your application's ``Gemfile``.
+
+#. Create a ``config/mongoid.yml`` configuration file and specify your
+   connection target.
+
+#. Create an application file and load your configuration file.
+
+#. Create {+odm+} models.
+
+Rails Application
+-----------------
+
+You can add {+odm+} to an existing Rails application to run alongside
+other ActiveRecord adapters. To use a combination of adapters, you
+can add the ``mongoid`` dependency and populate the configuration file
+with your connection information to start using MongoDB in your
+application.
+
+To adapt an existing Rails application to use only {+odm+} instead of
+ActiveRecord, you must make other configuration changes, as
+described in the following sections.
+
+Modify Dependencies
+~~~~~~~~~~~~~~~~~~~
+
+Add the ``mongoid`` gem to your application's ``Gemfile``:
+
+.. code-block:: ruby
+   :caption: Gemfile
+   
+   gem 'mongoid'
+
+To use {+odm+} as the *only* database adapter, remove or comment out any
+RDBMS libraries such as ``sqlite`` or ``pg`` listed in the ``Gemfile``.
+
+Then, install the dependencies by running the following command:
+
+.. code-block:: sh
+   
+   bundle install
+
+{+odm+} Configuration
+~~~~~~~~~~~~~~~~~~~~~
+
+Generate the default {+odm+} configuration by running the following
+command:
+
+.. code-block:: sh
+   
+   bin/rails g mongoid:config
+
+This generator creates the ``config/mongoid.yml`` configuration file
+used to configure the connection to the MongoDB deployment and the
+``config/initializers/mongoid.rb`` initializer file that you can use
+to set other options.
+
+In the ``config/mongoid.yml`` file, specify your connection string and
+other connection options.
+
+Modify Frameworks
+~~~~~~~~~~~~~~~~~
+
+Open the ``config/application.rb`` file and examine the contents. If it
+requires all the components of Rails by including the ``require
+'rails/all'`` specification, change this to specify individual
+frameworks.
+
+To verify the contents of ``rails/all`` for Rails 7, see the
+:github:`rails GitHub repository
+</rails/rails/blob/7-0-stable/railties/lib/rails/all.rb>`.
+
+The following code is a sample ``config/application.rb`` file that
+demonstrates how to specify individual frameworks instead of using ``rails/all``:
+
+.. code-block:: ruby
+   :caption: config/application.rb
+
+   # Remove or comment out rails/all
+   #require "rails/all"
+ 
+   # Add the following instead of rails/all:
+   require "rails"
+ 
+   # Comment out unneeded frameworks
```

Comment:
solid idea to show how to do this in code. if this is a comprehensive list, and if it isn't too burdensome to keep current, it might be good to direct them here from the opening paragraph ("...as shown in the following code example:")

Reply:
I'm not sure I totally understand what you mean. This list isn't necessarily what a user would need in their application to use Mongoid, it just demonstrates how to load frameworks individually.